trailblazer/store/crud/read.py

```diff
@@ -253,8 +253,13 @@ def _paginate_analyses(self, analyses: Query, request: AnalysesRequest) -> Query
             page_size=request.page_size,
         )
 
-    def get_analyses_being_uploaded(self) -> list[Analysis]:
+    def get_fastq_analyses_being_uploaded(self) -> list[Analysis]:
         return apply_analysis_filter(
-            filter_functions=[AnalysisFilter.BY_NOT_UPLOADED, AnalysisFilter.BY_COMPLETED],
+            filter_functions=[
+                AnalysisFilter.BY_NOT_UPLOADED,
+                AnalysisFilter.BY_COMPLETED,
+                AnalysisFilter.BY_WORKFLOW,
+            ],
             analyses=self.get_query(Analysis),
+            workflow=Workflow.FASTQ,
         ).all()
```

Comment:
can we make this a general function that receives a workflow?

Reply:
Sure 👍 https://github.com/Clinical-Genomics/trailblazer/pull/469/commits/8a0a8b125d3e38518604b8bc6879b8c6f453951b